### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DenverM80**
Update sample code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,93 +164,59 @@ The following section contains several examples of using the DS3 C SDK.  The fir
...
-#include <sys/stat.h>
-// "ds3.h" is the only header that needs to be included to use the DS3 API.
-#include "ds3.h"
-
-int main (int args, char * argv[]) {
-    // Allocate space for the response pointer.
-    // The DS3 client library will end up consuming this.
-    ds3_get_service_response *response; 
-    uint64_t i;
+int main (void) {
```
Comment (rpmoore) [inline README.md]: Why did you remove all the includes?

Comment (DenverM80) [inline README.md]: oops, I did not mean to remove all of them, will fix
